fitbenchmarking/utils/options.py

```diff
@@ -368,3 +367,30 @@ def read_list(s):
     :rtype: list of str
     """
     return str(s).split('\n')
+
+
+def read_range(s):
+    """
+    Utility function to allow ranges to be read by the config parser
+
+    :param s: string to convert to a list
+    :type s: string
+
+    :return: two element list [lower_lim, upper lim]
+    :rtype: list
+    """
+
+    if s[0] != '[' or s[-1] != ']':
+        raise ValueError("cmap_range specified without [] parentheses.")
+    rng = [float(item) for item in s[1:-1].split(",")]
+    if len(rng) != 2:
+        raise ValueError("cmap_range not specified with 2 elements.")
+    if rng[0] > rng[1]:
+        raise ValueError("Incorrect element order;"
+                         "cmap_range[0] > cmap_range[1] detected."
+                         "The elements must satisfy "
+                         "cmap_range[0] < cmap_range[1].")
+    if rng[0] < 0 or rng[0] > 1 or rng[1] < 0 or rng[1] > 1:
+        raise ValueError("One or more elements in cmap_range are "
+                         "outside of the permitted range 0 <= a <= 1.")
```

Comment:
These messages should probably be more generic so that the range can be used for other options if we need them in the future. That being said, this can be changed when/if we want to add those options.

Reply:
I can replace them with one message that just states the correct format for the time being. Something like: `cmap_range only accepts the following format [a, b], where a < b, a <= 1, b <=1`. Perhaps it's still not that generic but it would require less lines and be less to change if you want to repurpose `cmap_range`.

Reply:
I was thinking something more like removing the "cmap" bit, that should be handled by the exception handling in read_value

Reply:
I see what you mean now, I have made this change in my latest push.